front/components/agent_builder/submitAgentBuilderForm.ts

```diff
@@ -278,6 +277,41 @@ export async function submitAgentBuilderForm({
 
     const agentConfiguration = result.agentConfiguration;
 
+    // Export agent configuration to YAML
+    if (!isDraft) {
```

Comment:
For the sake of the example I added this condition not to end up with lots of created yaml files.